### DISCUSSION TITLE
PHP 8 upgrade/support

### DISCUSSION THREAD

**Discussion by roljohntorralba** [wprig/wprig#794] [Q&A] [answered]
Hi how do I configure wprig to support PHP 8?

I tried running npm scripts while on PHP 8 but it never gets past through gulp styles.

I tried reinstalling composer and npm libraries while on PHP 8 but with no luck.

Edit: clarification

**Comment by drikfan** [comment]
I ran into issues with wp 5.9 and php 8 as well. For me it was the phpcs in the gulp styles. I commeneted out that processing and it runs fine - not sure whats going on exactly as I havent had time to take a deeper look:
more info here:
https://github.com/wprig/wprig/discussions/786#discussioncomment-2068682

**Comment by dthenley** [comment]
@roljohntorralba where are you getting stuck at? I am able to run it on PHP 8 just fine at the moment.

**Comment by robruiz** [comment]
@drikfan @roljohntorralba  are you guys by change on windows? Need to know more about your system. There are a lot of variables. I noticed some possibly problematic code in gulp that might run weird on Windows, but I'm not currently certain.

**Reply by roljohntorralba** [reply]
Yes I was on Windows before, I'm on a Mac now, but with the same issue. My wp-rig project version is 2.0.1.

I've already edited a ton of files. Is there any way to update to the latest version safely?

**Reply by drikfan** [reply]
@robruiz I'm on the latest version of osx on an m1 Mac. I was able to use and am still able to use everything as I noted above.

**Comment by robruiz** [comment, marked as answer]
This issue has been addressed in a recent update to WP Rig. The issue was PHP Codesniffer. We added a new dependency straight to squizlabs package. Check composer.json for the update. Be sure to run "composer install" and "composer update" after you add this. Composer.lock needs to update. Then you should be ready to rock and roll. Let me know if you have any questions or further support.
